### PR TITLE
reformat all clojure namespaces

### LINKED ISSUE #4107: Reformat project code according to community style guide
### Description

Often new PR contains formatting changes (code indent) which are not related to the new functionality, but have to be committed because the editor auto formats the code. This heavily distracts reviewers and takes more time to review the actual changes in PR.

### Solution

Reformat code according to https://github.com/bbatsov/clojure-style-guide
Use Cursive/Emacs "Reformat code" to recursively format all the code in status-react. 

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/bbatsov/clojure-style-guide">bbatsov/clojure-style-guide</a></strong></div><div>clojure-style-guide - A community coding style guide for the Clojure programming language</div></blockquote>

### PR DESCRIPTION
**Pull request by rasom**
Fixes #4107
status: ready

### CODE REVIEW

File: src/status_im/chat/events/webview_bridge.cljs [hunk 1]
```diff
@@ -8,29 +8,29 @@
...
+                                    #(callback {:event :write-tag :card %})
+                                    #(callback {:event :write-tag :error %})))
+        :default))))
 
 (re-frame/reg-fx
- ::send-to-bridge
- (fn [[webview-bridge data]]
-   (when webview-bridge
-     (.sendToBridge webview-bridge data))))
+  ::send-to-bridge
```
Comment (rasom) [inline src/status_im/chat/events/webview_bridge.cljs]: this actually looks wrong

File: src/status_im/android/core.cljs [hunk 2]
```diff
@@ -38,35 +38,35 @@
...
-         ;; TODO(oskarth): Background click_action handler
-         (notifications/on-notification))
-       :component-will-unmount
-       (fn []
-         (.stop react/http-bridge)
-         (.removeEventListener react/app-state "change" app-state-change-handler))
-       :display-name "root"
-       :reagent-render views/main})))
+     {:component-will-mount
+                      (fn []
```
Comment (rasom) [inline src/status_im/android/core.cljs]: i'm not sure what we want to do with this code btw

File: src/status_im/bots/events.cljs [hunk 3]
```diff
@@ -88,19 +88,19 @@
 ;;;; Handlers
 
 (handlers/register-handler-fx
-  :set-in-bot-db
-  [re-frame/trim-v]
-  (fn [{:keys [db]} [params]]
-    (set-in-bot-db db params)))
+ :set-in-bot-db
```
Comment (yenda) [inline src/status_im/bots/events.cljs:91]: shouldn't these be 2 spaces according to https://github.com/bbatsov/clojure-style-guide#source-code-layout--organization ? That's what my clojure-mode auto-indents to as well

Comment (rasom) [reply src/status_im/bots/events.cljs:91]: fixed

Comment (rasom) [reply src/status_im/bots/events.cljs:91]: @yenda is it possible to run auto-indents for the whole project on your side just to see what is the difference?

Comment (dmitryn) [reply src/status_im/bots/events.cljs:91]: @rasom you could check with https://github.com/weavejester/cljfmt 
I believe cider-format-buffer uses same formatting rules.

Comment (yenda) [reply src/status_im/bots/events.cljs:91]: I am not aware of an automated way I think I have to open and save each file one by one. I'll give 10 min of googling to see if there is some emacs magic to do that.

Comment (yenda) [reply src/status_im/bots/events.cljs:91]: I am not aware of an automated way I think I have to open and save each file one by one. I'll give 10 min of googling to see if there is some emacs magic to do that.

Comment (dmitryn) [reply src/status_im/bots/events.cljs:91]: @rasom just confirmed with cider author that it uses `cljfmt` internally, so we could run `lein cljfmt fix src/*`

Comment (rasom) [reply src/status_im/bots/events.cljs:91]: ok, i'll try this, thanks

Comment (rasom) [reply src/status_im/bots/events.cljs:91]: @yenda btw this is a function call so we should have only one space here

**Comment by rasom** [comment]
ok so... too many changes, absolutely impossible to review. So I will split this to smaller PRs

**Pull request closed without merging**